### PR TITLE
Update JS collector config --> OTLP exporter config

### PR DESCRIPTION
**Pull request by cartermp**
The way you do it isn't really specific to collectors, so this can be generally useful.

---

Preview: https://deploy-preview-988--opentelemetry.netlify.app/docs/instrumentation/js/exporters/

### CODE REVIEW

File: content/en/docs/instrumentation/js/exporters.md [hunk 1]
```diff
@@ -7,10 +7,34 @@ In order to visualize and analyze your traces and metrics, you will need to expo
...
 
+- [OTLP endpoint or Collector](#otlp-endpoint-or-collector)
 - [Jaeger](#jaeger)
 - [Zipkin](#zipkin)
 - [Prometheus](#prometheus)
-- [OpenTelemetry Collector](#opentelemetry-collector)
+
+## OTLP endpoint or Collector
+
+To send trace data to a Collector you'll want to use an exporter package (e.g., `@opentelemetry/exporter-trace-otlp-http`):
```
Comment (chalin) [inline content/en/docs/instrumentation/js/exporters.md:17]: Let's avoid using "e.g." in the docs (yes I know there are other occurrences, but let's avoid introducing new ones).

Maybe:

> To send trace data ... package, such as `@opentelemetry/exporter-trace-otlp-http`.

File: content/en/docs/instrumentation/js/exporters.md [hunk 2]
```diff
@@ -7,10 +7,34 @@ In order to visualize and analyze your traces and metrics, you will need to expo
...
+
+```shell
+npm install --save @opentelemetry/exporter-trace-otlp-http
```
Comment (chalin) [inline content/en/docs/instrumentation/js/exporters.md:20]: I've started an effort to standardize all of these to use the `console` block-code language specifier, and start the command with the `$` prompt.

This is an optional change, I'll be sweeping the docs to change it elsewhere, so it's up to you if you want to do it now.

File: content/en/docs/instrumentation/js/exporters.md [hunk 3]
```diff
@@ -7,10 +7,34 @@ In order to visualize and analyze your traces and metrics, you will need to expo
...
+```
+
+And configure the exporter to point at and endpoint.
```
Comment (chalin) [inline content/en/docs/instrumentation/js/exporters.md:23]: And --> Also

File: content/en/docs/instrumentation/js/exporters.md [hunk 4]
```diff
@@ -7,10 +7,34 @@ In order to visualize and analyze your traces and metrics, you will need to expo
...
+const { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
+const { SimpleSpanProcessor } = require('@opentelemetry/tracing');
+
+const exporter = new OTLPTraceExporter({
+  url: '<your-collector-endpoint>/v1/traces', // optional - url default value is http://localhost:55681/v1/traces
+  headers: {}, // optional - collection of custom headers to be sent with each request
+});
+```
+
+It's recommended to use environment variables to set values like headers and an endpoint URL.
```
Comment (chalin) [inline content/en/docs/instrumentation/js/exporters.md:37]: How about just:

> Use environment variables to set ...

**Review by chalin — COMMENTED** [review]
A few minor comments, but otherwise LGTM

**Comment by chalin** [comment]
CC @open-telemetry/javascript-approvers

**Review by chalin — APPROVED** [review]
LGTM

**Comment by chalin** [comment]
@austinlparker - I'll let you have a final look and merge. Thx

**Review by austinlparker — APPROVED** [review]

**Pull request merged**